build/windows/installer/livenessprobe/livenessprobe.cpp

```diff
@@ -26,7 +26,7 @@ bool IsProcessRunning(const wchar_t *const executableName)
     PROCESSENTRY32 entry;
     entry.dwSize = sizeof(PROCESSENTRY32);
 
-    const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, NULL);
+    const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
```

Comment:
is this change needed

Reply:
> is this change needed

yes, there were warnings in the build base. 0 is the right parameter value to pass rather than NULL though NULL is 0.